src/css/footer.css

```diff
@@ -0,0 +1,68 @@
+.tagline-logo {
+  font-family: 'Impact', sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 100%;
+  letter-spacing: 0.05em;
+  text-transform: uppercase;
+}
+
+.logo-web {
+  color: var(--accent-green);
+}
+
+.footer {
+  padding-top: 40px;
+  padding-bottom: 24px;
+  line-height: 150%;
+  letter-spacing: 0.03em;
+  color: var(--black);
+}
+
+.footer-tumb {
+  display: block;
+}
+
+.footer-soc-list {
+  padding-top: 32px;
+  padding-bottom: 40px;
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.footer-soc-item:hover {
+  color: var(--accent-green);
+}
+
+.footer-copyright {
+  margin: auto;
+  width: 197px;
```

Comment:
In this case it is not a critical, but you should use property `max-width` instead fix width by `width` property